src/openag_brain/software_modules/sensor_persistence.py

```diff
@@ -27,6 +27,7 @@
 
 from openag_brain import params
 from openag_brain.util import resolve_message_type
```

Comment:
We should remove resolve_message_type if not used. (would also be good to check that it is unused elsewhere in the codebase + remove from utils)